CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
```

Comment:
There are some utility tools which are useful to analyze the SSA and AST representation before writing rules or analyser. 

For instance to dump the SSA, the `ssadump` tool can be used as following:
```
 ssadump -build F main.go
 ```
 It has various flags combinations to print different values.

Also for AST, we have a utility tool in https://github.com/securego/gosec/blob/master/cmd/gosecutil/tools.go which can be compiled and used as standalone.

```
gosecutil -tool ast main.go
````

I would maybe add a section to mention this.

Reply:
Added!